### DISCUSSION TITLE
Lightest debian base image?

### DISCUSSION THREAD

**Discussion by smartaquarius10** [dotnet/dotnet-docker#4678] [General]
Can someone suggest the lightest debian type base image for containers. 

I build self contained .net application in bullseye slim but it is making 374 mb of file even with PublishTrimmed=true. 

I cant use alpine because I wanted to add gss-ntlmssp package. Please help.

**Comment by mthalman** [comment]
I would suggest trying out the [Ubuntu chiseled containers](https://devblogs.microsoft.com/dotnet/dotnet-6-is-now-in-ubuntu-2204/#net-in-chiseled-ubuntu-containers). These images are still in preview but expect them to be supported later this year.

**Reply by smartaquarius10** [reply]
Hey @richlander , Hope you are doing well. Do we have any tentative date available for these ubuntu chiseled images. When can we use them in production. 

Looking forward to your reply. Thank you. 

Kind Regards,
Tanul

**Reply by smartaquarius10** [reply]
I hope they support gss-ntlmssp package. I’ll check that tomorrow morning.
